lib/urlwatch/reporters.py

```diff
@@ -735,3 +741,43 @@ def submit(self):
         else:
             logger.debug('Not formatting as Markdown; dependency on markdown2 not met?')
             client_api.send_message(room_id, body_markdown)
+
+
+class XMPPReporter(TextReporter):
+    """Custom XMPP Reporter"""
```

Comment:
I noticed that we have "Custom $SOMETHING Reporter" in many existing reporters, but I now changed this in the master branch.

Maybe "Send a message using the XMPP Protocol"